icc-x-api/icc-bekmehr-x-api.ts

```diff
@@ -23,14 +23,7 @@ export class IccBekmehrXApi extends iccBekmehrApi {
     const auth = this.headers.find(h => h.header === "Authorization")
     this.wssHost = new URL(this.host, window.location.href).href
       .replace(/^http/, "ws")
-      .replace(/:\/\//, "://" + (auth ? atob(auth.data.replace(/Basic /, "")) + "@" : ""))
```

Comment:
Correction needed to comply with Chrome limitation. The identification is handled by the sessionId specified below

Reply:
Not an issue in MS.